src/osdep/osdep.c

```diff
@@ -71,6 +71,19 @@
 #include "util/memory.h"
 #include "util/string.h"
 
+/* This is used in WIN32 to differenciate between
+ * file and socket. In case it's a tcp socket it has
+ * treated differently */
+
+#ifdef WIN32
+int
+is_socket(LPVOID s)
```

Comment:
I found out the elinks.exe does work in wine but fails on real iron. The problem was that unix-like systems are handling file sockets, terminal sockets and tcp/ip sockets all the same. But on Windows the tcp/ip socket has to be handled differntly while using the same function.